archiveFactSheets/archive.py

```diff
@@ -1,45 +1,138 @@
+# -*- coding: utf-8 -*-
+"""Script for archiving the specified factsheets in an input file.
+
+This script allows the user to archive factsheets from their workspace.
+The factsheets specified in the given input file will be set to ARCHIVED
+upon successful completion of the script. This script also uses cli inputs
+to gather all the necessary information.
+
+Example:
+    $ python archiveFactsheets.py
+
+Attributes:
```

Comment:
In Python we usually use the term Global variables to describe variables used in the context of constants.
```suggestion
Global variables:
```